package.json

```diff
@@ -38,7 +38,7 @@
   "devDependencies": {
     "@types/debug": "4",
     "@types/mocha": "^5.2.7",
-    "@types/node": "^12.12.17",
+    "@types/node": "^14.0.20",
```

Comment:
I don't think it's a good idea to pursue 14 version of Node.
Official node site recommends 12.19.0 LTS version at the moment.

Reply:
freeSockets were added in v0.11.4 https://nodejs.org/api/http.html#http_agent_freesockets
There's no need to update node to 14.